bears/general/LineCountBear.py

```diff
@@ -18,24 +19,58 @@ def _get_blank_line_count(self, file):
         return num_blank_lines
 
     def run(self, filename, file, max_lines_per_file: int,
-            exclude_blank_lines: bool=False):
+            ignore_files_regex: typed_list(str),
+            exclude_blank_lines: bool=False,
```

Comment:
Change its default value to True ;P

Reply:
btw blank lines as in trailing blank lines?

Reply:
@newbazz Exclude_blank_lines was written by someone else. This is part of code which already was written here when I started doing this issue. :) So I wouldn't want change this :)

Reply:
@newbazz And blank lines are allow by default, so this should be set to True :)